client/post-editor/post-editor.jsx

```diff
@@ -43,7 +42,6 @@ import {
 	getEditorPostId,
 	getEditorPath,
 	isConfirmationSidebarEnabled,
-	isEditorOnlyRouteInHistory,
```

Comment:
I believe that this selector was written for this case, and isn't used anywhere else... should we also remove the selector and its tests?

Reply:
I wasn't sure whether or not we should remove it, but since it's not currently being used, it can always be put back later. I'll remove.